src/stores/GuildMemberRoleStore.js

```diff
@@ -20,7 +20,8 @@ class GuildMemberRoleStore extends Collection {
    * @private
    */
   get _filtered() {
-    return this.guild.roles.filter(role => this.member._roles.includes(role.id));
+    const everyone = this.guild.roles.get(this.guild.id);
```

Comment:
Or you could use `Guild#defaultRole`.

Reply:
Done! I was looking for something like `GuildRoleStore#default` (aka `guild.roles.default`) in the code, but `Guild#roles` is a `RoleStore`, and `GuildRoleStore` itself doesn't exist. Since we have `roles.highest` (instead of `highestRole`) for example, being the former more OOP, I had assumed `defaultRole` was changed to `roles.default` but it's not 😄